src/components/molecules/Supplier/AddSupplierData/index.tsx

```diff
@@ -11,10 +11,10 @@ import {
   WrapItem,
 } from '@chakra-ui/react';
 import { useForm } from 'react-hook-form';
-import AddSupplierButtons from 'components/Supplier/AddSupplierButtons';
-import breakpoints from 'styled/Supplier/AppSupplierData/breakpoints';
-import breakpointsFontSize from 'styled/Supplier/AppSupplierData/breakpointsFontSize';
-import breakpointsMargin from 'styled/Supplier/AppSupplierData/breakpointsMargin';
+import AddSupplierButtons from '../AddSupplierButtons/index';
```

Comment:
@dominikmlynarczyk said that we should be using absolute paths (just from `/src` directory).
Please apply it in all appropriate places so I don't have to write the same comment everywhere.

Reply:
It will be corrected